src/main/webapp/jsp/feedbackSubmissionEdit.jsp

```diff
@@ -134,7 +134,16 @@
                         %>
                         </select>
                     </div>
-                    <div <%=(question.isRecipientNameHidden()) ? "class=\"col-sm-12\"" : "class=\"col-sm-10\""%>>
+                    <div 
+                    <% if (question.isRecipientNameHidden()) {
+                    	   out.println("class=\"col-sm-12\""); 
+                       } else if(numOfResponseBoxes == maxResponsesPossible) {
+                           out.println("class=\"col-sm-10\"");
+                       } else {   
+                    	   out.println("class=\"col-sm-8\""); 
+                       } 
+                    %>>
```

Comment:
This block of code seems to be repeated below. 
There is a loop at line 80 iterating through the questions, and the class can be stored in a variable there.
This can prevent the repeat in code(and prevent running multiple times for the same question), and using a variable will make the code look cleaner.